ansible/playbook/openstack/openstack_vm_create_passwordstore.yml

```diff
@@ -81,8 +86,12 @@
 
   post_tasks:
     - name: "DON'T FORGET TO SECURE YOUR SERVER"
-      debug:
-        msg: "Trying to start start server securization automatically For manual execution: $ ansible-playbook ansible/playbook/sec_host.yml -e vm_name={{ vm_name }} -e provider=openstack"
+      ansible.builtin.debug:
+        msg: 
+          - "DON'T FORGET TO SECURE YOUR SERVER!!!"
```

Comment:
Is this step still needed on RHOS ?

Reply:
Even though these VM are inside the RedHat VPN, I think we should keep them safer anyway, as a precaution.

Reply:
Can you then add a link to the doc explaining what we will do part of the VM securisation ?

Reply:
Added a link in the OpenStack README to the README having the securization information.